Source/WebCore/platform/graphics/coreimage/FilterImageCoreImage.mm

```diff
@@ -61,12 +61,12 @@
     if (m_imageBuffer)
         return m_imageBuffer.get();
 
-    m_imageBuffer = IOSurfaceImageBuffer::create(m_absoluteImageRect.size(), 1, m_colorSpace, PixelFormat::BGRA8, RenderingPurpose::Unspecified);
+    m_imageBuffer = ImageBuffer::create<ImageBufferIOSurfaceBackend>(m_absoluteImageRect.size(), 1, m_colorSpace, PixelFormat::BGRA8, RenderingPurpose::Unspecified, { });
     if (!m_imageBuffer)
         return nullptr;
 
     auto destRect = FloatRect { FloatPoint(), m_absoluteImageRect.size() };
-    [sharedCIContext().get() render: m_ciImage.get() toIOSurface: downcast<IOSurfaceImageBuffer>(*m_imageBuffer).surface().surface() bounds:destRect colorSpace:m_colorSpace.platformColorSpace()];
+    [sharedCIContext().get() render: m_ciImage.get() toIOSurface: m_imageBuffer->surface()->surface() bounds:destRect colorSpace:m_colorSpace.platformColorSpace()];
```

Comment:
If `ImageBuffer::create()` returns a valid `ImageBuffer`, it is guaranteed the IOSurface was created. But maybe it will be clearer if we add an assertion saying this before this statement.